### DISCUSSION TITLE
Website is unreadable

### DISCUSSION THREAD

**Discussion by gebannt** [opengaming/osgameclones#1430] [General]
![image](https://user-images.githubusercontent.com/25573380/87455542-e78af080-c605-11ea-8523-f877b15d73c6.png)

This is a total mess and not readable at all.

**Comment by foobar13372** [comment]
Totally agree.

**Comment by pehsa** [comment]
Tried some new design option, for example flexbox:
4 column default
![image](https://user-images.githubusercontent.com/30371056/92467660-4453f300-f1d2-11ea-9861-d4a599b2cd92.png)
2 column for smaller display
![image](https://user-images.githubusercontent.com/30371056/92467713-5b92e080-f1d2-11ea-94ba-b0ea57696586.png)
1 column for small/mobile display
![image](https://user-images.githubusercontent.com/30371056/92467820-85e49e00-f1d2-11ea-8371-896bb89eb745.png)

I will change the design to be more good looking.
The games will be on little boxes. The game information/badge section will be hide default and just like the gallery it will be clicked if you want to know more information.
Maybe, if the game has images, then display the first image as "thumbnail" size.

Any other idea?

**Comment by cxong** [comment]
I don't think the layout is the biggest problem. There are too many tags which makes it very messy to read.
How about hiding most of the tags by default, and adding options to show more? E.g. by default, show only `type`, `status` and `development`.

**Comment by cxong** [comment]
The page has now been split into two sections: a main listing with reduced tags, and game pages listing only one game + its clones with the full set of tags.

I think the main listing should show as few tags as possible, but still be useful for players and developers, by showing tags that are make or break:

- For players, the most important tags are type and status
- For developers, development, language, framework and license
